<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Распределение объектов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="imgTab">
       <attribute name="title">
        <string>Изображение</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Данные</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="attrTab">
       <attribute name="title">
        <string>Атрибуты</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_hist">
       <attribute name="title">
        <string>Гистограммы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>555</width>
             <height>423</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <widget class="QWidget" name="page">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QCustomPlot" name="widget" native="true">
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2"/>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openImgAct"/>
    <addaction name="openDataAct"/>
    <addaction name="openAtrAc"/>
    <addaction name="separator"/>
    <addaction name="closeAct"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="file_dock">
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>157</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Файл</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCommandLinkButton" name="openDataBut">
       <property name="text">
        <string>Открыть изображение</string>
       </property>
       <property name="icon">
        <iconset resource="sources.qrc">
         <normaloff>:/icons/open_image_ic.png</normaloff>:/icons/open_image_ic.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="openImgBut">
       <property name="text">
        <string>Открыть данные</string>
       </property>
       <property name="icon">
        <iconset resource="sources.qrc">
         <normaloff>:/icons/add_param_ic.png</normaloff>:/icons/add_param_ic.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="openAttrBut">
       <property name="text">
        <string>Открыть атрибуты</string>
       </property>
       <property name="icon">
        <iconset resource="sources.qrc">
         <normaloff>:/icons/view_groups_objects_ic.png</normaloff>:/icons/view_groups_objects_ic.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="filter_dock">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Создать фильтр</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Группы</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <action name="openDataAct">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/add_param_ic.png</normaloff>:/icons/add_param_ic.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл данных</string>
   </property>
   <property name="toolTip">
    <string>Файл с координатами объектов и их контурами</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="openImgAct">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/open_image_ic.png</normaloff>:/icons/open_image_ic.png</iconset>
   </property>
   <property name="text">
    <string>Открыть изображение</string>
   </property>
   <property name="toolTip">
    <string>Файл самого изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="openAtrAc">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/view_groups_objects_ic.png</normaloff>:/icons/view_groups_objects_ic.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл атрибутов</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="closeAct">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/icons/delete_groups_object_ic.png</normaloff>:/icons/delete_groups_object_ic.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="toolTip">
    <string>Закрыть все открытые файлы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>external/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
